scss/_list-group.scss

```diff
@@ -197,8 +197,21 @@
 //
 // Add modifier classes to change text and background color on individual items.
 // Organizationally, this must come after the `:hover` states.
-@each $color, $value in $background-colors {
-  @include list-group-item-variant($color, $value, color-contrast($value));
+@each $state, $value in $background-colors { // Boosted mod: instead of `@each $state in map-keys($theme-colors)`
+  .list-group-item-#{$state} {
+    --#{$prefix}list-group-color: #{color-contrast($value)}; // Boosted mod: instead of `var(--#{$prefix}#{state}-text)`
+    --#{$prefix}list-group-bg: #{$value}; // Boosted mod: instead of `var(--#{$prefix}#{$state}-bg-subtle)`
+    // Boosted mod: no `--#{$prefix}list-group-border-color`
+
+    &.list-group-item-action {
+      &:hover { // Boosted mod: no `&:focus`
```

Comment:
Need to have this line ? since the CSS var should already change only the hover state by default. Same on BS

Reply:
See #37888.